third_party/xla_client/xla_computation_client.cc

```diff
@@ -232,23 +235,35 @@ string XlaComputationClient::GetEffectiveDevice(const string& device) const {
   return device;
 }
 
-void XlaComputationClient::FlushReleasedHandles() {
-  std::lock_guard<std::mutex> lock(lock_);
-  size_t num_handles = released_handles_.size();
-  if (num_handles > 0) {
-    metrics::TimedSection timed(ReleaseHandlesTimeMetric());
-    std::vector<std::unique_ptr<GlobalData>> released_handles;
-    released_handles.swap(released_handles_);
-    GlobalData::Release(std::move(released_handles));
-    ReleaseHandlesMetric()->AddSample(num_handles);
-  }
-}
-
 void XlaComputationClient::ReleaseXlaData(XlaData* xla_data) {
   std::lock_guard<std::mutex> lock(lock_);
   released_handles_.push_back(xla_data->Release());
 }
 
+void XlaComputationClient::StartHandleReleaser() {
+  std::thread releaser([this]() { HandleReleaser(); });
+  releaser.detach();
+}
+
+void XlaComputationClient::HandleReleaser() {
+  auto period = std::chrono::milliseconds(500);
+  for (;;) {
```

Comment:
Don't we prefer `while(true)`?

Reply:
Is it?
I have not found anything in the guide or the ToW, but I do not have a particular attachment with nay of the two styles :)

Reply:
I'd go with `while (true)`.

Reply:
Alright